examples/AccountManagement/AcceptServiceLinks.php

```diff
@@ -94,46 +104,94 @@ public static function main()
+            PHP_EOL
+        );
         foreach ($response->getMerchantCenterLinks() as $merchantCenterLink) {
             /** @var MerchantCenterLink $merchantCenterLink */
-            if ($merchantCenterLink->getStatus() === MerchantCenterLinkStatus::PENDING) {
-                // Enables the pending link.
-                $merchantCenterLinkToUpdate = new MerchantCenterLink([
-                    'resource_name' => $merchantCenterLink->getResourceName(),
-                    'status' => MerchantCenterLinkStatus::ENABLED
-                ]);
-
-                // Constructs an operation that will update the merchant center link,
-                // using the FieldMasks utility to derive the update mask. This mask tells the
-                // Google Ads API which attributes of the merchant center link you want to change.
-                $merchantCenterLinkOperation = new MerchantCenterLinkOperation();
-                $merchantCenterLinkOperation->setUpdate($merchantCenterLinkToUpdate);
-                $merchantCenterLinkOperation->setUpdateMask(
-                    FieldMasks::allSetFieldsOf($merchantCenterLinkToUpdate)
-                );
-
-                // Issues a mutate request to update the merchant center link and prints some
-                // information.
-                $response = $merchantCenterLinkServiceClient->mutateMerchantCenterLink(
-                    $customerId,
-                    $merchantCenterLinkOperation
-                );
-                printf(
-                    "Enabled a Merchant Center Link with resource name '%s' to the Google Ads "
-                    . "account '%s'.%s",
-                    $response->getResult()->getResourceName(),
+            printf(
+                "Link '%s' has status '%s'.%s",
+                $merchantCenterLink->getResourceName(),
+                MerchantCenterLinkStatus::name($merchantCenterLink->getStatus()),
+                PHP_EOL
+            );
+            // Approves a pending link request for a Google Ads account with the specified customer
+            // ID from a Merchant Center account with the specified merchant center account ID.
+            if (
+                $merchantCenterLink->getIdUnwrapped() === $merchantCenterAccountId
+                && $merchantCenterLink->getStatus() === MerchantCenterLinkStatus::PENDING
+            ) {
+                // Updates the status of Merchant Center link to 'ENABLED' to approve the link.
+                self::updateMerchantCenterLinkStatus(
+                    $merchantCenterLinkServiceClient,
                     $customerId,
-                    PHP_EOL
+                    $merchantCenterLink,
+                    MerchantCenterLinkStatus::ENABLED
                 );
+                // There is only one MerchantCenterLink object for a given Google Ads account and
+                // Merchant Center account, so we can break early.
+                break;
             }
         }
```

Comment:
Add a message in case no pending link for the given CID & Merchant could be found

Reply:
Although not very perfect, the above message showed this case too by saying that "0 links found".